src/main.c

```diff
@@ -194,17 +195,19 @@ float time_of_day()
 
 float get_daylight()
 {
-    float timer = time_of_day();
-    if (timer < 0.5)
-    {
-        float t = (timer - 0.25) * 100;
-        return 1 / (1 + powf(2, -t));
-    }
-    else
-    {
-        float t = (timer - 0.85) * 100;
-        return 1 - 1 / (1 + powf(2, -t));
-    }
+	/**
```

Comment:
You will want to add doxygen documentation for this section of code as well with references to the requirement the code pertains to.

Reply:
@Jonathan-1996-Projects I have added the Doxygen documentation